ddp-workspace/projects/ddp-sdk/src/lib/components/address/addressEmbedded.component.ts

```diff
@@ -257,6 +263,17 @@ export class AddressEmbeddedComponent implements OnDestroy, OnInit {
         this.initializeComponentState();
     }
 
+    private setupScrollToErrorAction(): void {
+        this.scroll_up$ = this.validationRequested$.pipe(
+            map(validationRequested => validationRequested && this.block.scrollTo),
+            debounceTime(300)
+        );
+    }
+    scrollUpExecutedAction(executed: boolean): void {
+        this.block.scrollTo=!executed;
```

Comment:
We always set `this.block.scrollTo=` in this case (after a page was scrolled up to the failed question/component).
No need to pass a boolean argument here.

Reply:
done